fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java

```diff
@@ -778,12 +778,7 @@ public String visitAnalyticExpr(AnalyticExpr node, Void context) {
         }
 
         public String visitArrayExpr(ArrayExpr node, Void context) {
-            boolean explicitType = node.isExplicitType();
-
```

Comment:
Why remove this?

Reply:
just used on explain array, I think we don't need output type info on `explain`? and explain verbose was support the feature